wolfcrypt/src/wc_port.c

```diff
@@ -1845,6 +1846,10 @@ int wolfSSL_CryptHwMutexUnLock(void) {
         return 0;
     }
 
+#elif defined(WOLFSSL_USER_MUTEX)
+
+    /* Use user own mutex */
```

Comment:
Can you add some comments to show the mutex function templates the user would need to implement in their code? Looks like `wc_InitMutex`, `wc_LockMutex`, `wc_UnLockMutex` and `wc_FreeMutex`.

Reply:
Added coding template